### ISSUE TITLE
[FEAT]: A Divider between two tabs.

### ISSUE THREAD

**Issue by NOTtanuki** [Tormak9970/TabMaster#207]
Basically a simple divider for the tabs to make visual groups, you are able to KIND OF make one in TabMaster but:
1. I make the filters so obscure that no games pop up on the "divider" tab.
2. When scrolling tabs between divider, I have to scroll over it, instead of going to the next time.
Heres an example of the concept:
![image](https://github.com/user-attachments/assets/c1f88273-f0a9-4f9f-811b-90c378703c55)
TLDR: Make a native divider and make it unselectable/unscrollable when displayed
Extra: Have a setting for how much padding on divider maybe?

**Comment by Tormak9970** [comment]
I'm not sure how feasible this is, as it would be hard to specify where the divider is since we make tabs reorderable, but we'll look into it

**Comment by Tormak9970** [comment]
We haven't found a good solution due to how the tabs are rendered, so I'm closing this for now. We may revisit it in the future
